internal/pkg/cli/storage_init.go

```diff
@@ -46,15 +47,88 @@ We'll deploy the Cloudformation for the storage when you run 'svc deploy'.`
 
 var fmtStorageInitCreateConfirm = "Okay, we'll create %s %s named %s linked to your %s service."
 
+var fmtStorageInitDynamoInfoText = `We're going to ask you some questions about your database schema and datatypes.
+You can change any of this information at the end by editing the CloudFormation
+generated at ./ecs-project/%s/addons/%s-%s.yml.`
```

Comment:
Do we use this variable? Maybe we should have the path as fmt verb "%s" so that it outputs the correct path if the command is run within a subdirectory in the workspace or on a different platform.